L10nSnapshotTests/L10nSuite2SnapshotTests.swift

```diff
@@ -123,17 +123,27 @@ class L10nSuite2SnapshotTests: L10nBaseSnapshotTests {
         // First time only: The message "Your passwords are now protected
         // by Face ID..." is present when the Firefox app is run after the
         // simulator has been erased.
-        waitForExistence(app.navigationBars.element(boundBy: 0), timeout: 3)
-        waitForExistence(app.otherElements.buttons.element(boundBy: 2))
-        app.otherElements.buttons.element(boundBy: 2).tap()
+        // The biometric screen is supposed to appear here.
+        // https://github.com/mozilla-mobile/firefox-ios/issues/15642
+        // waitForExistence(app.navigationBars.element(boundBy: 0), timeout: 3)
+        // waitForExistence(app.otherElements.buttons.element(boundBy: 2))
+        // app.otherElements.buttons.element(boundBy: 2).tap()
 
         let passcodeInput = springboard.secureTextFields.firstMatch
         waitForExistence(passcodeInput, timeout: 30)
         passcodeInput.tap()
         passcodeInput.typeText("foo\n")
 
+        // The biometric screen is not supposed to be here during the first run.
+        // This snippet is here as a workaround.
+        // https://github.com/mozilla-mobile/firefox-ios/issues/15642
+        waitForNoExistence(passcodeInput)
+        if app.otherElements.buttons.staticTexts.element(boundBy: 5).exists {
+            app.otherElements.buttons.staticTexts.element(boundBy: 5).tap()
```

Comment:
Actually this does not work for me, it clicks on the previous button. 
![Simulator Screenshot - iPhone 14 - 2023-08-01 at 11 13 12](https://github.com/mozilla-mobile/firefox-ios/assets/1897507/db9dd23d-51c3-4fb4-8859-ed76619361c3)


Using 6 works though (I'm using iPhone 14, iOS 16.4)

Reply:
The index `5` works for me even on `es` locale. 🤔 

Note that this test does not work on `ru` and `bn` so far. In this locale, the biometric screen is not shown after the passcode is enetered.

I have updated the `l10n-screenshots.sh` script that iOS 16.4 is used.

Reply:
This works for me from command line, so we are fine here. We would need to add a `snapshot("Biometric-screen")` so that we see that screen translated here too before L142